Solutions/CohesitySecurity/Tests/helios.py

```diff
@@ -1,12 +1,47 @@
 #!/usr/bin/env python3
 
 import requests
+import datetime
+import datetime
+import time
+
+
+def get_alerts_details(alert_ids, apiKey):
+    maxAlerts = len(alert_ids)
+    alert_ids = ",".join(alert_ids)
+    api_url = "https://helios.cohesity.com/mcm/alerts?maxAlerts=" + str(maxAlerts) + "&alertIdList=" + alert_ids
+    headers = {"Content-Type": "application/json"}
+    headers["authority"] = "helios.cohesity.com"
+    headers["apiKey"] = apiKey
+    response = requests.get(api_url, headers=headers)
+    return response.json() if response.json() else None
 
 
 def get_alert_details(alert_id, apiKey):
```

Comment:
Please remove this function and let the users always use always use the newly created function. We can remove the duplicated code that way.

Reply:
done